template/README.md

```diff
@@ -674,6 +675,22 @@ import { expect } from 'chai';
 
 and then use them in your tests like you normally do.
 
+### Initializing Test Environment
+
```

Comment:
Please add “>Note: this feature is available since `react-scripts@0.4.0`” here. You can copy and paste an existing similar notes and change the version.